src/graph/node_initializers.cpp

```diff
@@ -254,7 +254,7 @@ template Ptr<NodeInitializer> range<IndexType>(IndexType begin, IndexType end, I
 }  // namespace marian
 
 
-#if BLAS_FOUND
+#if 0
```

Comment:
I assume this will be fixed up for integration

Reply:
Yes, let's not resolve this conversation until it has been done later on.